### ISSUE TITLE
Lidarr states that my library directory is unavailable

### ISSUE THREAD

**Issue by vendion** [lidarr/Lidarr#1167]
**Describe the bug**
While trying to setup Lidarr I added the directory where my music library is stored but Lidarr has it flagged as "unavailable".

**To Reproduce**
1. Go to 'Settings'
2. Click on 'Media Management'
3. Scroll down to 'Root Folders'

**Expected behavior**
The music directory is added to where Lidarr can scan and import my current library.

**Screenshots**
![2020-06-09-225938_1674x154_scrot](https://user-images.githubusercontent.com/145018/84222184-f4677200-aaa4-11ea-82fc-17b372cb89be.png)

**Logs**
[Lidarr.txt](https://github.com/lidarr/Lidarr/files/4755955/Lidarr.txt)

**System info (please complete the following information):**
Version 0.7.1.1381
Mono Version 5.10.1.57
OS FreeBSD 12.1

**Comment by vendion** [comment]
As a work around if I create a symlink from _/media/music_ -> _/music_ it seems to be happy then although this is not preferable to me.

**Comment by opie4624** [comment]
The symlink trick doesn't seem to work for me.

**Comment by ta264** [comment]
I don't think this is a lidarr issue, I think you just have your jail set up wrong.

**Comment by vendion** [comment]
@ta264 Do you care to elaborate on that? I ask because I also have Sonarr (the project that Lidarr is based on) and Radarr (another Sonarr fork but for movies) running in the same Jail and they can access the respective directories just fine.

```
drwxrwxrwx    4 nobody  nobody     4B Jun  7  2019 downloaded
drwxrwxrwx   81 nobody  nobody    81B Aug 10 01:59 movies <- Radarr set to use this directory with no issues
drwxrwxrwx   80 nobody  nobody    80B Aug 25 21:44 music <- Lidarr should use this directory but says that it is unavailable for what ever reason
drwxrwxrwx  108 nobody  nobody   114B Aug 25 22:34 tv <- Sonarr set to use this directory with no issues
```

All three directories is also mounted into the Jail the same way, but yet this works for me at least

```
lrwxrwxrwx  1 root  wheel    12B Jun  9 23:06 /music -> /media/music
```

**Comment by opie4624** [comment]
@vendion Looks like this is fixed in `nightly` #969 commit 3743b023f2e3105309d8101df03d61813edf398e
